3.7/programs-starter-architecture.md

```diff
@@ -93,6 +93,109 @@ It contains the following files & sub-directories.
   - `arangod.log`: The log file of the server
   - `arangod_command.txt`: File containing the exact command line of the started server (for debugging purposes only)
 
+## Starter configuration file
+
+The Starter can be configured using a configuration file. The format of the
+configuration file is the same as the `arangod` configuration file format.
+For more details, refer to the [General Configuration of ArangoDB programs and tools](administration-configuration.html#configuration-file-format).
+
+The default configuration file of the Starter is `arangodb-starter.conf`.
+It can be changed using the `--configuration` option. 
+For more information about supported commands and other
```

Comment:
1) 'supported commands' are not related to configuration file, I think it's better remove it from this sentence.
2) I think it is better to refer to `docs/3.10/programs-starter-options.html` page here for a list of options.